odd-platform-api/src/test/java/org/opendatadiscovery/oddplatform/repository/NamespaceRepositoryImplTest.java

```diff
@@ -0,0 +1,62 @@
+package org.opendatadiscovery.oddplatform.repository;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Optional;
+import org.junit.jupiter.api.Test;
+import org.opendatadiscovery.oddplatform.BaseRepositoryTest;
+import org.opendatadiscovery.oddplatform.misc.TestDataUtils;
+import org.opendatadiscovery.oddplatform.model.tables.pojos.NamespacePojo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.jdbc.Sql;
+
+/**
+ * Integration test for {@link NamespaceRepository}
+ *
+ * @author matmalik on 10.12.2021
+ */
+class NamespaceRepositoryImplTest extends BaseRepositoryTest {
+
+    @Autowired
+    private NamespaceRepository namespaceRepository;
+
+    TestDataUtils testDataUtils = new TestDataUtils();
+
+    /**
+     * Test case: Save {@link NamespacePojo} into db
+     * Expected result: {@link NamespacePojo} successfully saved into db
+     */
+    @Test
+    @Sql({"/scripts/dropAll.sql", "/scripts/createAll.sql"})
```

Comment:
You don't need to put this before each test method. `Sql` has a default execution phase `BEFORE_TEST_METHOD`